src/libsync/syncfilestatustracker.cpp

```diff
@@ -116,11 +167,13 @@ void SyncFileStatusTracker::slotAboutToPropagate(SyncFileItemVector& items)
     foreach (const SyncFileItemPtr &item, items) {
         // qDebug() << Q_FUNC_INFO << "Investigating" << item->destination() << item->_status;
 
-        if (showErrorInSocketApi(*item))
+        if (showErrorInSocketApi(*item)) {
             _syncProblems[item->_file] = SyncFileStatus::StatusError;
-        else if (showWarningInSocketApi(*item))
+        } else if (showWarningInSocketApi(*item)) {
             _syncProblems[item->_file] = SyncFileStatus::StatusWarning;
-
+        } else if( showSyncInSocketApi(*item)) {
+            _syncProblems[item->_file] = SyncFileStatus::StatusSync;
```

Comment:
i think _syncProblems should be only for errors and warning.  Any reason to put this in?